api-docs/openapi-diff.ts

```diff
@@ -0,0 +1,37 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+const jsonDiff = require('json-diff');
+const https = require('https');
```

Comment:
let's convert to import statements

Reply:
hmm lemme know if i'm missing something here.

I'm having difficulty figuring out how to get node evalutaing this as an es module rather than commonjs :( 
sorry i'm a bit rusty in this are... any pointers?

Reply:
Actually - i finally got it working.
Not sure what i was doing before to make it unhappy 😅